### DISCUSSION TITLE
Show and Tell: A place to share cool applications, imagery, etc. that involves PING-Mapper

### DISCUSSION THREAD

**Discussion by CameronBodine** [CameronBodine/PINGMapper#76] [Show and tell]
Hi PING-Mapper community,
I'm creating this discussion page for anyone to share neat things they have done with PING-Mapper. This can be sonar images, comparisons with other workflows, or anything else you would like to share. I also use Twitter ([@bodine_cs](https://twitter.com/bodine_cs)) and LinkedIn ([cameronbodine](https://www.linkedin.com/in/cameronbodine)) to share updates. Feel free to follow me there, or tag me in any posts you make!

**Comment by AdrianPinchbeck** [comment]
**Sidescan Ghosts!**

Halfway through post-processing some Humminbird recordings of an area of mixed ground using PINGMapper and QGIS, something suddenly caught my eye while merging tiles…

I had previously spent a morning mapping a small No Take Zone just off the east coast of England, teaming up a standalone Helix 9 setup, running off a 7Ah alarm battery, with a home-made towfish on a transducer extension cable.  A stiff breeze blowing off the land on my starboard side made it tricky to maintain track against an opposing ebbing tide and along a carefully arranged grid of waypoints created earlier.  Consequently, I wasn’t really studying the Humminbird screen for any detail, relying on my other navigation electronics hard-wired on the boat, only periodically checking towfish altitude above some areas of pretty rough ground and boulders until I was done.

Back at the computer I processed each separately recorded mile long track using PINGMapper, taking account of both layback (approximated using high school trigonometry) and cross-track error, and then started to process each track on QGIS to start creating a complete mosaic of the zone.  Then that something caught my eye, a strange line running along a clean area of smooth sand.  
 
![NTZGroundMappingTest — QGIS 22_08_2023 17_20_30 (2)](https://github.com/CameronBodine/PINGMapper/assets/121804798/b5844bf3-b722-404f-bae6-a7ecae4fd553)

As I zoomed in to investigate further, the line continued in both directions, with periodic lumps along the way.  It was clearly a string of around eight pots in an area I didn't expect.  

![NTZGroundMappingTest — QGIS 17_08_2023 11_56_31 (2)](https://github.com/CameronBodine/PINGMapper/assets/121804798/a9f1a11c-b43b-4f42-86bd-8979a511f7a0)

There’s a large crab and lobster fishery in this part of the North Sea, but this was well inside the No Take Zone, so there shouldn’t be any fishing gear in this position at all.  Further, I would have spotted any surface marker, had the string had one, and quickly lifted the towfish clear of any potential snags.  I hadn’t seen anything, so what was going on?

A quick email to one of the local wreck dive teams with a list of WGS84 numbers for each pot, and they then confirmed that a dive would be made, so the wait for feedback began.  Three weeks later, an email from the divers popped up on my feed.  All pots had been located, but no anchor or ends were found.  Maybe they were just part of a string broken up and relocated in a storm?  Pots had clearly been there for some time, at they were found half buried in sand, but the divers opened all hatch doors and various previously doomed critters were set free (pots will continue fishing indefinitely even without any bait).

The next job is to get permission for a commercial potter to access the restricted area, and clear the string out of the No Take Zone using a pot hauler for good.

**Reply by CameronBodine** [reply]
Great story! Thanks for sharing Adrian.

That imagery looks really great despite the wind. Ideal conditions really. And great work spotting the line! Something like that can really be tough to spot. I'm glad that PINGMapper is helping to fight the good fight of locating ghost fishing gear!

**Reply by AdrianPinchbeck** [reply]
Thanks Cameron, I've been really impressed with the image quality produced with PINGMapper.  Unfortunately our area of sea is rarely flat, so keeping the transducer running squarely through clean water below the boat is imperative.  Spotting the ghost gear was just a bonus, but I'm hoping to be able to measure areas of exposed reef and other features made visible. The mapping is to record any habitat changes over time using repeatable waypoints, as warming seas may well bring us some surprises in the future!
